libraries/botbuilder-ai/botbuilder/ai/luis/luis_recognizer_v3.py

```diff
@@ -0,0 +1,176 @@
+import aiohttp
+import asyncio
+import json
+
+from typing import Dict
+from .luis_recognizer_internal import LuisRecognizerInternal
+from .luis_recognizer_options_v3 import LuisRecognizerOptionsV3
+from .luis_application import LuisApplication
+
+from botbuilder.core import (
+    RecognizerResult,
+    TurnContext,
+)
+from .activity_util import ActivityUtil
+
+
+class LuisRecognizerV3(LuisRecognizerInternal):
+    _dateSubtypes = ["date", "daterange", "datetime", "datetimerange", "duration", "set", "time", "timerange"]
+    _geographySubtypes = ["poi", "city", "countryRegion", "continent", "state"]
```

Comment:
As a side note, JS uses the value "continet", which is probably incorrect as "continent" seems the appropriate value.  Should open an issue on that repo.

Reply:
Python is right. Im working on PR to JS right now